sway-types/src/span.rs

```diff
@@ -10,6 +10,13 @@ pub struct Span {
 }
 
 impl Span {
+    pub fn empty() -> Self {
+        Span {
+            span: pest::Span::new(" ".into(), 0, 0).unwrap(),
+            path: None,
+        }
+    }
+
```

Comment:
We've needed this for a while!  I wonder if instead of a literally empty span we should use a message like "Source span not available." just to avoid weird error messages or something.

Reply:
Good idea!